modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPreloader.java

```diff
@@ -331,6 +347,15 @@ private IgniteCheckedException stopError() {
             }
         }
 
+        if (! skippedPartitionsLackHistSupplier.isEmpty())
+            log.info("Unable to perform historical rebalance cause " +
+                "history supplier is not available [parts=" + S.compact(skippedPartitionsLackHistSupplier) +
+                ", topVer=" + topVer + ']');
```

Comment:
Redundant whitespace.
Missed check log.isInfoEnabled.
Missed bracket for multiline statement.
Missed group id and name in logged string.